AmericanMedicalAssociation/ama-style-guide-2
# EWL-6527 homepage membership block spacing and margins

## JIRA Ticket(s)
- [EWL-6527: Homepage Member Feature Updates
](https://issues.ama-assn.org/browse/EWL-6527)

## Description:
Desktop | Reduce spacing between 3 vertical stubs on the right side of the component
Desktop, Tablet | The gray background of div.ama_member-feature-rail should expand further behind the image/content of div.ama_hero-story (see screenshots, Zeplin, tablet Zeplin)
Mobile | Two of the Article Stubs in div.ama__member-feature-rail should appear on mobile, stacked below as seen in the mobile Zeplin

## To Test:
- switch your SG2 branch to `bugfix/EWL-6499-membership-needs-more-sepacing`
- `gulp serve`
- enable local SG2 in your D8 in your local.yml 
- do a `scripts/build` outside your vagrant
- visit http://ama-one.local/
- `drush @one.local cr`
- observe the spacing around the membership block matches zeplin
desktop: https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5b313db4a1cee6697422dd24
mobile: https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5b32bd0adf510ffb4747954d

## Automated Test:
n/a

## Style Guide:


## Relevant Screenshots/GIFs:
<img width="1281" alt="screen shot 2018-11-05 at 2 20 01 pm" src="https://user-images.githubusercontent.com/2271747/48024290-e651b280-e105-11e8-89f2-58bc3309beb7.png">
<img width="484" alt="screen shot 2018-11-05 at 2 20 21 pm" src="https://user-images.githubusercontent.com/2271747/48024311-f2d60b00-e105-11e8-99b6-8ce38ef259bf.png">


## Additional Notes:
n/a

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
